modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -42,12 +42,14 @@ def validate_veteran_identifiers(require_birls: false) # rubocop:disable Metrics
           )
         end
 
-        ClaimsApi::Logger.log('validate_identifiers',
+        ClaimsApi::Logger.log('validate_identifiers_before_MPI_ADD_REPSONSE',
                               rid: request.request_id,
                               require_birls: require_birls,
                               header_request: header_request?,
                               ptcpnt_id: target_veteran.participant_id.present?,
-                              birls_id: target_veteran.birls_id.present?)
+                              birls_id: target_veteran.birls_id.present?,
+                              icn: target_veteran&.mpi_icn,
+                              time_stamp: Time.now.to_i)
```

Comment:
we get timestamps in the logs via Grafana. I'm not sure if we need this as well.